src/main/resources/bidder-config/connatix.yaml

```diff
@@ -13,11 +13,11 @@ adapters:
     usersync:
       cookie-family-name: connatix
       iframe:
-        url: "https://capi.connatix.com/us/pixel?pId=53&gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&us_privacy={{us_privacy}}&callback={{redirect_url}}"
+        url: "https://capi.connatix.com/us/pixel?pId=53&gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&us_privacy={{us_privacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&callback={{redirect_url}}"
```

Comment:
Please update the macro to a PBS-Java flavour. For both urls.
From this:
```&gpp={{.GPP}}&gpp_sid={{.GPPSID}}```

To this:
```&gpp={{gpp}}&gpp_sid={{gpp_sid}}```

Reply:
thank you! addressed!